ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/ImageMessage.qml

```diff
@@ -31,19 +31,24 @@ Rectangle {
         }
 
         Image {
+            // This minimum height is for the initial load of the chat to work correctly
+            // Without this, the chat doesn't understand that the image wil lhave a height and doesn't scroll to the bottom
+            property int minHeight: 400
             id: imageMessage
             anchors.horizontalCenter: parent.horizontalCenter
             anchors.top: (index == 0) ? parent.top: parent.children[index-1].bottom
             anchors.topMargin: imageChatBox.chatVerticalPadding
             sourceSize.width: imageChatBox.imageWidth
             source: modelData
             onStatusChanged: {
-                if (imageMessage.status == Image.Error) {
+                if (imageMessage.status === Image.Error) {
                     imageMessage.height = 0
+                    imageMessage.minHeight = 0
+                    imageMessage.source = ""
                     imageMessage.visible = false
                     imageChatBox.height = 0
                     imageChatBox.visible = false
-                } else if (imageMessage.status == Image.Ready) {
+                } else if (imageMessage.status === Image.Ready) {
                     messageItem.scrollToBottom(true, messageItem)
```

Comment:
Do we not need to update the `minHeight` property here? Could we possibly use the [`Image.paintedHeight`](https://doc.qt.io/qt-5/qml-qtquick-image.html#paintedHeight-prop) property?

Reply:
Nope, `minHeight` is only used if the height is 0